### PR TITLE
fixed the background issue of the dcamera preview on C2

### PR DESCRIPTION
**Pull request by deanlee**
the background need to be cleared before display dcam frame on C2.

**before:**

**after:**

### CODE REVIEW

File: selfdrive/ui/qt/widgets/cameraview.cc [hunk 1]
```diff
@@ -199,12 +199,13 @@ void CameraViewWidget::updateFrameMat(int w, int h) {
 }
 
 void CameraViewWidget::paintGL() {
-  if (latest_texture_id == -1) {
+  if (latest_texture_id == -1 || stream_type == VISION_STREAM_RGB_FRONT) {
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/widgets/cameraview.cc:202]: this isn't a real way to fix the issue. how expensive is it to always do this?

Comment (deanlee) [reply selfdrive/ui/qt/widgets/cameraview.cc:202]: It's very cheap and doesn't even make a difference

Comment (deanlee) [reply selfdrive/ui/qt/widgets/cameraview.cc:202]: We can only clear the background every frame, otherwise there will be problems on C2

Comment (deanlee) [reply selfdrive/ui/qt/widgets/cameraview.cc:202]: It should be caused by  by the difference between c2's GraphicBuffer/visonbuf and other devices

Comment (adeebshihadeh) [reply selfdrive/ui/qt/widgets/cameraview.cc]: I don't understand. is it not cheap to always glClearColor and glClear on all platforms for all cameras?

Comment (deanlee) [reply selfdrive/ui/qt/widgets/cameraview.cc]: it's same on all platforms. think we should always call glclear before drawing to avoid weird artifacts

**Pull request merged**